cmd/raidz_test/raidz_bench.c

```diff
@@ -83,8 +83,17 @@ run_gen_bench_impl(const char *impl)
 			/* create suitable raidz_map */
 			ncols = rto_opts.rto_dcols + fn + 1;
 			zio_bench.io_size = 1ULL << ds;
-			rm_bench = vdev_raidz_map_alloc(&zio_bench,
-			    BENCH_ASHIFT, ncols, fn+1);
+
```

Comment:
That's right.  These changes originated with the raidz expansion project.  While not strictly necessary for dRAID I wanted to keep a version of them since it allows us to the `raidz_bench` test to validate the correctness of the multi-row map changes.